app/models/team.rb

```diff
@@ -9,6 +9,7 @@
 #  description   :text
 #  kind_of_sport :string
 #  private       :boolean
+#  avatar_data   :text
```

Comment:
Is there a difference between `avatar_data` and `:avatar`?

Reply:
Yes